tests/atlocal.in

```diff
@@ -27,7 +29,7 @@ else
     exit
 fi
 
-RPMTEST="@CMAKE_CURRENT_BINARY_DIR@/testing"
+RPMTEST=$RPMTREE
```

Comment:
This is missing quotes, which might matter in some exotic cases.

Reply:
That said, the quoting is likely inconsistent to begin with, this just caught my eye as the previous assignment had them.

Reply:
Ugh, you're right. Not that I wasn't aware, but I'm kinda sloppy when it comes to quoting in my own changes. I just consider quotes too noisy to bother :facepalm: Which, arguably, isn't a good idea :smile: 

And yup, it's inconsistent already, but that's no excuse to continue that. I'll fix this, thanks!

Reply:
Oh I know the feeling. Guess why they're so inconsistent to begin with? :laughing: